addons/mail/static/tests/emoji/emoji.test.js

```diff
@@ -122,6 +124,21 @@ test("recent category (basic)", async () => {
     });
 });
 
+test("search emojis should include recent emojis, making them more prominent", async () => {
+    const pyEnv = await startServer();
+    const channelId = pyEnv["discuss.channel"].create({ name: "" });
+    await start();
+    await openDiscuss(channelId);
+    await click("button[aria-label='Emojis']");
+    await contains(".o-EmojiPicker-navbar [title='Frequently used']", { count: 0 });
+    await click(".o-EmojiPicker-content .o-Emoji", { text: "🤥" });
+    await click("button[aria-label='Emojis']");
+    await insertText("input[placeholder='Search for an emoji']", "lie");
+    // Ensure that "🤥" is the first emoji in the search results
+    await animationFrame();
+    expect(queryAll(".o-EmojiPicker-content .o-Emoji")[0].textContent).toBe("🤥");
```

Comment:
Can `await contains(".o-EmojiPicker-sectionIcon", { count: 0 });` rather than wait for `animationFrame()`, the former is more reliable and end-user actually see the change on UI